src/hidpp20.c

```diff
@@ -2572,6 +2572,7 @@ hidpp20_buttons_to_cpu(struct hidpp20_device *device,
 			break;
 		case HIDPP20_BUTTON_SPECIAL:
 			button->special.special = b->special.special;
+			button->special.profile = b->special.profile;
```

Comment:
Can you expand the commit message a bit here? I think I know what it does but not having to guess would be preferable ;)

Remember not everyone has the specs or remembers what they say, so a good commit message can make having to look them up unnecessary.